### DISCUSSION TITLE
Importing files is inefficient

### DISCUSSION THREAD

**Discussion by 25pwn** [ppy/osu#20556] [Ideas] [answered]
Say I have a list of files I want to import. 
There are a number of ways to go about this:

If osu is not running, I can select a bunch of filles and `Open with` osu. However, only one of the files is imported this way.
If osu is running, the same does nothing. However, I can drag them into osu and all of them will import just fine.
Since firefox can open multiple files with `Open with`, it may also be achievable with osu.

Using the `Import files` menu, the things that slow you down are:
- You can only select one file at a time, so it takes θ(n) time.
- Importing each file requires clicking on the file, and clicking `Import` on the other side of the screen.
- After clicking import, if you click on the next file and a previous import finishes, the file will be unselected.
Therefore, you have to select, click import, wait and repeat.
Adding a way to `ctrl`/`shift` select files and adding a keyboard shortcut for `Import` would improve this.

**Comment by peppy** [comment, marked as answer]
You can drag many files into the game.

If osu! is running and you are trying to double-click files, you are likely hitting https://github.com/ppy/osu/issues/14824.
